modules/order/src/Entity/Order.php

```diff
@@ -463,6 +494,21 @@ public static function baseFieldDefinitions(EntityTypeInterface $entityType) {
       ->setLabel(t('Data'))
       ->setDescription(t('A serialized array of additional data.'));
 
+    $fields['billing_profile'] = BaseFieldDefinition::create('entity_reference')
+      ->setLabel(t('Billing profile'))
+      ->setDescription(t('Billing profile'))
+      ->setRequired(TRUE)
+      ->setSetting('target_type', 'profile')
+      ->setSetting('handler', 'default')
+      ->setTranslatable(TRUE)
```

Comment:
We also need a ->setSetting('handler_settings', ['target_bundles' => ['profile']]); This will limit the reference field to the appropriate bundle.